src/cwts/networkanalysis/run/RunNetworkClustering.java

```diff
@@ -161,11 +170,18 @@ public static void main(String[] args)
             String arg = args[argIndex];
             try
             {
-                if (arg.equals("-q") || arg.equals("--quality-function"))
+                if (arg.equals("-n") || arg.equals("--normalization"))
                 {
-                    if (((argIndex + 1) >= args.length) || (!args[argIndex + 1].equals("CPM") && !args[argIndex + 1].equals("modularity")))
-                        throw new IllegalArgumentException("Value must be 'CPM' or 'modularity'.");
-                    useModularity = args[argIndex + 1].equals("modularity");
+                    if (((argIndex + 1) >= args.length) || (!args[argIndex + 1].equals(NORMALIZATION_NAMES[NO_NORMALIZATION]) && !args[argIndex + 1].equals(NORMALIZATION_NAMES[ASSOCIATION_STRENGTH]) && !args[argIndex + 1].equals(NORMALIZATION_NAMES[FRACTIONALIZATION]) && !args[argIndex + 1].equals(NORMALIZATION_NAMES[MODULARITY])))
```

Comment:
Perhaps we can make a small helper function for checking whether a value is in an array, as it is not clearly readable in this way.